packages/fxa-auth-server/lib/log.js

```diff
@@ -134,6 +135,9 @@ Lug.prototype.notifyAttachedServices = function (name, request, data) {
         // potential timing issues
         data.ts = data.ts || Date.now() / 1000; // Convert to float seconds
 
+        // Tag all events with the issuing service.
+        data.iss = data.iss || ISSUER;
```

Comment:
Is `data.iss` ever going to be set in practice? Looks like not even the tests pass it in based on the diff.

Reply:
(not sure if that made sense, I meant set on the rhs, do we need/want `data.iss ||`?)

Reply:
You're right, I don't think we'll ever pass it in explicitly (and probably shouldn't).